src/math/Vector3.d.ts

```diff
@@ -211,7 +211,7 @@ export class Vector3 implements Vector {
 	/**
 	 * Sets this vector to cross product of itself and v.
 	 */
-	cross( a: Vector3, w?: Vector3 ): this;
+	cross( a: Vector3, w: Vector3 ): this;
```

Comment:
The idea was to remove `w: Vector3`^^.

Reply:
> The idea was to remove `w: Vector3`^^. You can do this for `add()`, too.

I think  `Vector3.d.ts#fromBufferAttribute()` can be removed the `offset`, it makes sense.